examples/docs_snippets/docs_snippets/guides/dagster/dagster_pipes/subprocess/with_asset_check/dagster_code.py

```diff
@@ -16,9 +16,15 @@ def no_empty_order_check(
         shutil.which("python"),
         dg.file_relative_path(__file__, "external_code.py"),
     ]
-    return pipes_subprocess_client.run(
+
+    results = pipes_subprocess_client.run(
         command=cmd, context=context.op_execution_context
-    ).get_asset_check_result()
+    ).get_results()
```

Comment:
Not sure if this is still the intended goal of the code. It is failing on `get_asset_check_result`